tests/testcases/Group3-RBAC/3-08-LDAP-define-harbor-admin-by-dn.md

```diff
@@ -0,0 +1,31 @@
+Test 3-08 - Define Harbor Admin By DN (LDAP Mode)
+=======
+
+# Purpose:
+
+To verify that harbor admin can be defined by LDAP group DN
+
+# References:
+User guide
+
+# Environment:
+* This test requires that a Harbor instance is running and available.
+* Harbor is set to authenticate against an LDAP or AD server. ( auth_mode is set to **ldap_auth** .) The user data is stored in an LDAP or AD server.
+* A linux host with Docker CLI installed (Docker client).
+* At least five(5) non-admin users are in Harbor. 
+
+# Test Steps:
+
+
+1. Create group harbor_sys_admin in LDAP.
+1. Create LDAP user sys_admin in LDAP, and add sys_admin to the member of group harbor_sys_admin.
+1. Login as admin user, set the Configuration: "LDAP Groups With Admin Privilege" with the DN of harbor_sys_admin.
```

Comment:
could you give the pull navigation path of "LDAP Groups With Admin Privilege" in UI?